tests/E2E/test/campaigns/common_scenarios/pages.js

```diff
@@ -175,67 +203,67 @@ module.exports = {
         });
       } else {
         test('should click on "Bulk actions > Delete selected" button', () => {
           return promise
+            .then(() => client.waitForVisibleAndClick(Pages.Category.bulk_actions_select_all_button))
             .then(() => client.waitForExistAndClick(Pages.Category.bulk_actions_button))
             .then(() => client.waitForExistAndClick(Pages.Category.bulk_actions_delete_button));
         });
         test('should accept the currently displayed alert dialog', () => client.alertAccept());
-        test('should verify the appearance of the green validation', () => client.checkTextValue(Pages.Common.success_panel, '×\nThe selection has been successfully deleted.'));
+        test('should verify the appearance of the green validation', () => client.checkTextValue(Pages.Common.success_panel, 'The selection has been successfully deleted.'));
       }
     }, 'design');
   },
   createAndPreviewPage: function (pageData, category = "", pageID = 1) {
     scenario('Create a CMS page and check the preview ', client => {
       test('should click on "Add new page" button', () => client.waitForExistAndClick(Pages.Page.add_new_page_button));
+      test('should set "title" input', () => client.waitAndSetValue(Pages.Page.title_input, pageData.meta_title + date_time));
       if (pageData.hasOwnProperty('page_category')) {
         test('should select the "Parent category" option ', () => {
           return promise
+            .then(() => client.waitForVisibleAndClick(Pages.Page.expand_button))
             .then(() => client.getAttributeInVar(Pages.Page.category_option.replace('%CATEGORY_NAME', pageData.page_category), "value", "categoryNameID"))
-            .then(() => client.waitAndSelectByValue(Pages.Page.page_category, global.tab['categoryNameID']))
+            .then(() => client.waitForVisibleAndClick(Pages.Page.page_category.replace('%V',global.tab['categoryNameID'])));
         });
       }
-      test('should set "Meta title" input', () => client.waitAndSetValue(Pages.Common.name_input, pageData.meta_title + date_time));
-      test('should set "Meta description" input', () => client.waitAndSetValue(Pages.Common.meta_description_input, pageData.meta_description));
+      test('should set "Meta title" input', () => client.waitAndSetValue(Pages.Page.meta_title_input, pageData.meta_title + date_time));
+      test('should set "Meta description" input', () => client.waitAndSetValue(Pages.Page.meta_description_input, pageData.meta_description));
       for (let i in pageData.meta_keyword) {
         test('should set the "Meta Keywords - ' + pageData.meta_keyword[i] + '" input', () => {
           return promise
-            .then(() => client.waitForExistAndClick(Pages.Page.meta_keywords_input))
-            .then(() => client.keys(pageData.meta_keyword[i]))
+            .then(() => client.waitAndSetValue(Pages.Page.meta_keywords_input,pageData.meta_keyword[i]))
+            //.then(() => client.keys())
```

Comment:
You can remove this line if you don't need it

Reply:
I forgot it, thanks.